src/Tests/Moryx.ControlSystem.Tests/EnumInstructionResultTests.cs

```diff
@@ -115,5 +116,25 @@ private class MyInput
         {
             public int Foo { get; set; }
         }
+
+        private enum TestResults6
+        {
+            [EnumInstruction, Display(Name = "Value 1")]
+            Value1,
+            [EnumInstruction("Value 2")]
+            Value2
+        }
+
+        [Test]
+        public void UsesDisplayResultsWhereFound()
+        {
+            // Act
+            var instructionResult = new EnumInstructionResult(typeof(TestResults6), result => { });
+
+            // Assert
+            Assert.AreEqual(2, instructionResult.Results.Count(), "There should be two results, because none are hidden");
+            Assert.AreEqual("Value 1", instructionResult.Results[0]);
+            Assert.AreEqual("Value 2", instructionResult.Results[1]);
```

Comment:
What is the intended behaviour if you only have the display attribute? That could be reflected well in this unit test I think 😊

Reply:
That is covered in the other tests, as `Display` does not affect the filtering of members, but I can add a test.